src/saveload/afterload.cpp

```diff
@@ -3123,6 +3123,12 @@ bool AfterLoadGame()
 		}
 	}
 
+	if (IsSavegameVersionBefore(SLV_FIX_NEUTRAL_STATION_DOCKING_TILE)) {
```

Comment:
```suggestion
	if (IsSavegameVersionBefore(SLV_FIX_NEUTRAL_STATION_DOCKING_TILE) && !IsSavegameVersionBefore(SLV_MULTITILE_DOCKS)) {
```
to not iterate all stations twice, as now the original `IsSavegameVersionBefore(SLV_MULTITILE_DOCKS)` block above this fix will be correct.

Reply:
Other option is to extract the iteration from `IsSavegameVersionBefore(SLV_MULTITILE_DOCKS)` block and run it unconditionally like `AfterLoadRoadStops()`, removing the need for the savegame bump